### PR TITLE
Updates for release v2022.2.1 (Zonemaster-LDNS)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Updates for release v2022.2.1.

## Changes

Updates Changes and LDNS.pm.

## How to test this PR

Review Changes and inspect LDNS.pm.

### CODE REVIEW

File: lib/Zonemaster/LDNS.pm [hunk 1]
```diff
@@ -2,7 +2,7 @@ package Zonemaster::LDNS;
 
 use 5.014;
 
-our $VERSION = '3.0.0';
+our $VERSION = '3.0.1';
```
Comment (tgreenx) [inline lib/Zonemaster/LDNS.pm:5]: 3.0.1 -> 3.1.0

File: Changes [hunk 2]
```diff
@@ -1,5 +1,11 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+3.0.1 2023-01-31 (public fix version)
+ [Fixes]
```
Comment (tgreenx) [inline Changes:4]: [Fixes] -> [Features]

**Review by tgreenx — COMMENTED** [review]
I see the changes introduced by #166 as more of a "Minor" change rather than a "Patch" change. Wouldn't you agree? If so, see my comments below.

**Comment by matsduf** [comment]
> I see the changes introduced by #166 as more of a "Minor" change rather than a "Patch" change. Wouldn't you agree? If so, see my comments below.

It is also tagged as "minor". I guess I would not paying enough attention. Thank your reviewing thoroughly.

File: lib/Zonemaster/LDNS.pm (hunk 1 above)
Comment (matsduf) [reply lib/Zonemaster/LDNS.pm]: Fixed.

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes]: Fixed.-

**Review by marc-vanderwal — APPROVED** [review]
The minor version bump is indeed appropriate. Looks good to me.

**Review by tgreenx — APPROVED** [review]

**Pull request merged**